docs/bundles/SyliusFixturesBundle/custom_fixture.rst

```diff
@@ -0,0 +1,128 @@
+Custom fixture
+==============
+
+Basic fixture
+-------------
+
+Let's create a fixture that loads all countries from ``Intl`` library. We'll extend on ``AbstractFixture`` in order
+to skip the configuration part for now:
+
+.. code-block:: php
+
+    namespace AppBundle\Fixture;
+
+    use Sylius\Bundle\FixturesBundle\Fixture\AbstractFixture;
+    use Sylius\Bundle\FixturesBundle\Fixture\FixtureInterface;
+
+    final class CountryFixture extends AbstractFixture implements FixtureInterface
+    {
+        private $countryManager;
+
+        public function __construct(ObjectManager $countryManager)
+        {
+            $this->countryManager = $countryManager;
+        }
+
+        public function getName()
+        {
+            return 'country';
+        }
+
+        public function load(array $options)
+        {
+            $countriesCodes = array_keys(\Intl::getRegionBundle()->getCountryNames());
+
+            foreach ($countriesCodes as $countryCode) {
+                $country = new Country($countryCode);
+
+                $this->countryManager->persist($country);
+            }
+
+            $this->countryManager->flush();
+        }
+    }
+
+The next step is to register this fixture:
+
+.. code-block:: yaml
+
+    <service id="app.fixture.country" class="AppBundle\Fixture\CountryFixture">
+        <argument type="service" id="doctrine.orm.entity_manager" />
+        <tag name="sylius_fixtures.fixture" />
+    </service>
+
+
+Fixture is already registered and ready to use in your suite:
+
+.. code-block:: yaml
+
+    sylius_fixtures:
+        suites:
+            my_suite:
+                fixtures:
+                    country: ~
+
+Configurable fixture
+--------------------
+
+Loading all countries may be useful, but what if you want to load some defined countries in one suite and all
+countries in the other? You don't need to create multiple fixtures, one configurable will do the job!
+
+.. code-block:: php
+
+    final class CountryFixture extends AbstractFixture implements FixtureInterface
+    {
+        // ...
+
+        public function load(array $options)
+        {
+            foreach ($options['countries'] as $countryCode) {
+                $country = new Country($countryCode);
+
+                $this->countryManager->persist($country);
+            }
+
+            $this->countryManager->flush();
+        }
+
+        protected function configureOptionsNode(ArrayNodeDefinition $optionsNode)
+        {
+            $optionsNodeBuilder
+                ->arrayNode('countries')
+                    ->performNoDeepMerging()
+                    ->defaultValue(array_keys(\Intl::getRegionBundle()->getCountryNames()))
+                    ->prototype('scalar')
+            ;
+        }
+    }
+
+.. note::
+
+    ``AbstractFixture`` implements ``ConfigurationInterface::getConfigTreeBuilder()`` and exposes handy
```

Comment:
**The**  ``AbstractFixture`` implements **the** ``ConfigurationInterface::getConfigTreeBuilder()`` method and exposes **a** handy...